addons/survey/models/survey_survey.py

```diff
@@ -560,129 +559,43 @@ def get_print_url(self):
-                    res[(input_line.matrix_row_id.id, input_line.suggested_answer_id.id)] += 1
-                if input_line.answer_type == 'char_box' and (not(current_filters) or input_line.user_input_id.id in current_filters):
-                    comments.append(input_line)
-            result_summary = {'answers': answers, 'rows': rows, 'result': res, 'comments': comments}
-
-        # Calculate and return statistics for text_box, char_box, date
-        if question.question_type in ['text_box', 'char_box', 'date', 'datetime']:
-            result_summary = []
-            for input_line in input_lines:
-                if not(current_filters) or input_line.user_input_id.id in current_filters:
-                    result_summary.append(input_line)
-
-        # Calculate and return statistics for numerical_box
-        if question.question_type == 'numerical_box':
-            result_summary = {'input_lines': []}
-            all_inputs = []
-            for input_line in input_lines:
-                if not(current_filters) or input_line.user_input_id.id in current_filters:
-                    all_inputs.append(input_line.value_numerical_box)
-                    result_summary['input_lines'].append(input_line)
-            if all_inputs:
-                result_summary.update({'average': round(sum(all_inputs) / len(all_inputs), 2),
-                                       'max': round(max(all_inputs), 2),
-                                       'min': round(min(all_inputs), 2),
-                                       'sum': sum(all_inputs),
-                                       'most_common': Counter(all_inputs).most_common(5)})
-        return result_summary
-
-    @api.model
-    def get_input_summary(self, question, current_filters=None):
-        """ Returns overall summary of question e.g. answered, skipped, total_inputs on basis of filter """
-        current_filters = current_filters if current_filters else []
-        result = {}
-        search_line_ids = current_filters if current_filters else question.user_input_line_ids.ids
-
-        result['answered'] = len([line for line in question.user_input_line_ids if line.user_input_id.state != 'new' and not line.user_input_id.test_entry and not line.skipped])
-        result['skipped'] = len([line for line in question.user_input_line_ids if line.user_input_id.state != 'new' and not line.user_input_id.test_entry and line.skipped])
-
-        return result
+                scoring_failed_count += count_data_item['scoring_success_count']
+
+        success_graph = json.dumps([{
+            'text': _('Passed'),
+            'count': scoring_success_count,
+            'color': '#2E7D32'
+        }, {
+            'text': _('Missed'),
+            'count': scoring_failed_count,
+            'color': '#C62828'
+        }])
+
+        total_quizz_passed = scoring_success_count + scoring_failed_count
```

Comment:
`total_quizz_passed` was referring to `quizz_passed` that doesn't exist anymore.
Also, I've learned since then that quiz is spelled with only one "z".

So `total_quiz_passed_succeeded` ?